tests/unittests/multimodal/test_clip_score.py

```diff
@@ -0,0 +1,88 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from collections import namedtuple
+from functools import partial
+
+import pytest
+import torch
+from transformers import CLIPModel as _CLIPModel
+from transformers import CLIPProcessor as _CLIPProcessor
+
+from torchmetrics.multimodal.clip_score import CLIPScore
+from unittests.helpers import seed_all
+from unittests.helpers.testers import MetricTester
+
+seed_all(42)
+
+
+Input = namedtuple("Input", ["images", "captions"])
+
+
+captions = [
+    "28-year-old chef found dead in San Francisco mall",
+    "A 28-year-old chef who recently moved to San Francisco was "
+    "found dead in the staircase of a local shopping center.",
+    "The victim's brother said he cannot imagine anyone who would want to harm him,\"Finally, it went uphill again at "
+    'him."',
+    "A lawyer says him .\nMoschetto, 54 and prosecutors say .\nAuthority abc Moschetto  .",
+]
+
+_random_input = Input(images=torch.randint(255, (2, 2, 3, 224, 224)), captions=[captions[0:2], captions[2:]])
+
+
+def _compare_fn(preds, target, version):
+    processor = _CLIPProcessor.from_pretrained(version)
+    model = _CLIPModel.from_pretrained(version)
+    inputs = processor(text=target, images=[p.cpu() for p in preds], return_tensors="pt", padding=True)
+    outputs = model(**inputs)
+    logits_per_image = outputs.logits_per_image
+    return logits_per_image.diag().mean().detach()
+
+
+@pytest.mark.parametrize("version", ["openai/clip-vit-base-patch32"])
+@pytest.mark.parametrize(
+    "input",
+    [
+        _random_input,
+    ],
+)
+class TestCLIPScore(MetricTester):
+    atol = 1e-5
+
+    @pytest.mark.parametrize("ddp", [True, False])
+    def test_clip_score(self, input, version, ddp):
+        # images are preds and targets are captions
+        preds, target = input
```

Comment:
```suggestion
    "input",
    [
        _random_input.preds, _random_input.target
    ],
)
class TestCLIPScore(MetricTester):
    atol = 1e-5

    @pytest.mark.parametrize("ddp", [True, False])
    def test_clip_score(self, preds, target, version, ddp):
        # images are preds and targets are captions
```

To stay consistent with other tests?

+ Shouldn't we add `dist_sync_on_step = True | False` into the configuration?